photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -131,6 +131,12 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
     "use warm start",
     "Whether to train the current model with coefficients initialized by the previous model.")
 
+  val coordinateInterceptIndices: Param[Map[CoordinateId, Int]] =
+    ParamUtils.createParam[Map[CoordinateId, Int]](
+    "coordinate intercept indices",
+    "A map of coordinate ID to intercept index, if enabled."
+  )
```

Comment:
Please move closing bracket onto the line above (137). Please also drop "if enabled" from the parameter description (it doesn't make sense, since there is a default value for it, thus it is always enabled).